docs/architecture/grpc-for-wcf-developers/protobuf-maps.md

```diff
@@ -1,11 +1,36 @@
 ---
 title: Protobuf maps for dictionaries - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: Using Protobuf maps to represent .NET's Dictionary types
 author: markrendle
-ms.date: 09/02/2019
+ms.date: 09/09/2019
 ---
 
 # Protobuf maps for dictionaries
 
+Protobuf's equivalent of the .NET `Dictionary<TKey, TValue>` type is `map<key_type, value_type>`.
+
+```protobuf
+message StockPrices {
+    map<string, double> prices = 1;
+}
+```
+
+In the generated code, `map` fields use the `Google.Protobuf.Collections.MapField<TKey, TValue>`. Again, this implements the standard .NET interfaces, including `IDictionary<TKey, TValue>`.
+
+Map fields cannot be `repeated`, but you can create a nested message containing a map and use `repeated` on the message type.
```

Comment:
this makes sense in English but perhaps the first repeated shouldn't be blocked from localization?